packages/query/src/block-processor.ts

```diff
@@ -29,6 +29,7 @@ import {
   TransactionInfo,
 } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/view/v1alpha1/view_pb';
 import { StateCommitment } from '@buf/penumbra-zone_penumbra.bufbuild_es/penumbra/crypto/tct/v1alpha1/tct_pb';
+import { computePositionId } from '@penumbra-zone/wasm-ts/src/dex';
```

Comment:
Let's add `export * from './src/dex';` to `packages/wasm/index.ts` and we can shorten the import like the others